luigi/worker.py

```diff
@@ -713,11 +713,25 @@ def _get_work_task_id(self, get_work_response):
         if get_work_response['task_id'] is not None:
             return get_work_response['task_id']
         elif 'batch_id' in get_work_response:
-            task = load_task(
-                module=get_work_response.get('task_module'),
-                task_name=get_work_response['task_family'],
-                params_str=get_work_response['task_params'],
-            )
+            try:
+                task = load_task(
+                    module=get_work_response.get('task_module'),
+                    task_name=get_work_response['task_family'],
+                    params_str=get_work_response['task_params'],
+                )
+            except Exception as ex:
```

Comment:
Since you catch such a broad exception, the code isn't self-documenting. I'm sure your use-case is fine, but can you add a comment within the `except` block of what typically goes wrong and makes us end up here?